### DISCUSSION TITLE
Export UI typo

### DISCUSSION THREAD

**Discussion by pravdomil** [keiwando/toonsquid#219] [Ideas]
GIF is abbreviation, so it should be written as all caps.

<img width="246" alt="image" src="https://user-images.githubusercontent.com/2387356/227634801-d05cedc3-d0c5-4ba7-81e5-885cb10f3941.png">

**Comment by Lioncub901** [comment]
You seemed to have typed a lot discussion threads. If you don’t mind could you make one discussion thread and put all you ideas there. You can edit the discussion thread to add more ideas. Doing so will make reading all your requests easier since they are not split up.

**Comment by pravdomil** [comment]
fixed thank you
